packages/server/test/unit/args_spec.coffee

```diff
@@ -204,9 +211,9 @@ describe "lib/util/args", ->
 
       ## make sure it works with both --env=foo=bar and --config foo=bar
       @obj = @setup(
+        "--config",
```

Comment:
ohh, this is the problem - `--config` has moved, but its list of arguments remains in place - causing all the problems!

Reply:
Yah, I thought my need for alphabetization may be the culprit, but I couldn’t see it 🤦‍♀️